great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -498,11 +499,19 @@ def get_domain_records(
                     )
                 )
             else:
-                if ignore_row_if != "neither":
+                if ignore_row_if not in ["neither", "never"]:
                     raise ValueError(
                         f'Unrecognized value of ignore_row_if ("{ignore_row_if}").'
                     )
 
+                if ignore_row_if == "never":
+                    warnings.warn(
+                        f"""The correct "no-action" value of the "ignore_row_if" directive for the column pair case is \
+"neither" (the use of "{ignore_row_if}" will be deprecated).  Please update code accordingly.
+""",
+                        DeprecationWarning,
+                    )
+
```

Comment:
Does this complete the `neither` vs `never` story for now? Did your previous PR get rolled into this one?

Reply:
@NathanFarmer The previous PR is going through Azure checks (and I want to merge that one first).